src/main/resources/templates/db_scripts/schema.json

```diff
@@ -413,6 +413,33 @@
           "tableAlias": "titles"
         }
       ]
+    },
+    {
+      "tableName": "reasons_for_closure",
+      "fromModuleVersion": "mod-orders-storage-11.0.0",
```

Comment:
The last mod-orders-storage version we've released is 9.0.0 https://github.com/folio-org/mod-orders-storage/releases/tag/v9.0.0. So the next version must be 10.0.0